registration-processor/post-processor/registration-processor-message-sender-stage/src/main/java/io/mosip/registration/processor/message/sender/util/StatusNotificationTypeMapUtil.java

```diff
@@ -40,6 +40,13 @@ public StatusNotificationTypeMapUtil() {
 	private static Map<NotificationStageStatus, NotificationTemplateType> statusMapper() {
 		statusMap.put(NotificationStageStatus.QUALITY_CHECK_FAILED, NotificationTemplateType.TECHNICAL_ISSUE);
 		statusMap.put(NotificationStageStatus.CMD_VALIDATION_FAILED, NotificationTemplateType.TECHNICAL_ISSUE);
+		statusMap.put(NotificationStageStatus.OPERATOR_VALIDATION_FAILED, NotificationTemplateType.TECHNICAL_ISSUE);
+		statusMap.put(NotificationStageStatus.SUPERVISOR_VALIDATION_FAILED, NotificationTemplateType.TECHNICAL_ISSUE);
+		statusMap.put(NotificationStageStatus.INTRODUCER_VALIDATION_FAILED, NotificationTemplateType.TECHNICAL_ISSUE);
```

Comment:
what is difference between failed and error ? In introducer case, we might use DUPLICATE_UIN